### PR TITLE
[13.0][MIG] website_snippet_big_button: Migration to 13.0

### PR DESCRIPTION
**Pull request by willytal**
Standard procedure

TT27594 @Tecnativa

### CODE REVIEW

**Comment by willytal** [comment]
cc @Tardo @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]
Functional test in runboat

File: website_snippet_big_button/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+#~ msgstr "Seguir buscando"
+
+#~ msgid "I want another thing"
+#~ msgstr "Quiero otra cosa"
+
+#~ msgid "I want this"
+#~ msgstr "Quiero esto"
+
+#~ msgid "Request budget"
+#~ msgstr "Solicitar presupuesto"
```
Comment (victoralmau) [inline website_snippet_big_button/i18n/es.po:59]:
```suggestion
```

File: website_snippet_big_button/i18n/fr.po [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+"<span class=\"h3 d-block\">Je veux cela</span>\n"
+"                            <i class=\"fa fa-thumbs-o-up fa-5x mt16 mb16\"/>"
+"\n"
+"                            <span class=\"d-block\">Budget demandé</span>"
+
+#~ msgid "Continue searching"
+#~ msgstr "Continuer la recherche"
+
+#~ msgid "I want another thing"
+#~ msgstr "Je veux autre chose"
```
Comment (victoralmau) [inline website_snippet_big_button/i18n/fr.po:53]:
```suggestion
```

File: website_snippet_big_button/i18n/pt_BR.po [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+#~ msgstr "Continue procurando"
+
+#~ msgid "I want another thing"
+#~ msgstr "Eu quero outra coisa"
+
+#~ msgid "I want this"
+#~ msgstr "Eu quero isso"
+
+#~ msgid "Request budget"
+#~ msgstr "Solicitar orçamento"
```
Comment (victoralmau) [inline website_snippet_big_button/i18n/pt_BR.po:61]:
```suggestion
```

File: website_snippet_big_button/i18n/sl.po [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+#~ msgstr "Nadaljuj z iskanjem"
+
+#~ msgid "I want another thing"
+#~ msgstr "Želim nekaj drugega"
+
+#~ msgid "I want this"
+#~ msgstr "Želim to"
+
+#~ msgid "Request budget"
+#~ msgstr "Zahtevaj proračun"
```
Comment (victoralmau) [inline website_snippet_big_button/i18n/sl.po:59]:
```suggestion
```

File: website_snippet_big_button/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2015-2016 Tecnativa - Jairo Llopis <jairo.llopis@tecnativa.com>
+# Copyright 2019 Tecnativa - Cristina Martin R.
+# Copyright 2022 Tecnativa - Guillermo Gallego <guillermo.gallego@tecnativa.com>
```
Comment (victoralmau) [inline website_snippet_big_button/__manifest__.py:3]:
```suggestion
# Copyright 2015-2016 Tecnativa - Jairo Llopis
# Copyright 2019 Tecnativa - Cristina Martin R.
# Copyright 2022 Tecnativa - Guillermo Gallego
```

File: website_snippet_big_button/templates/assets.xml [hunk 6]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- # Copyright 2015-2016 Tecnativa - Jairo Llopis <jairo.llopis@tecnativa.com>
```
Comment (victoralmau) [inline website_snippet_big_button/templates/assets.xml:2]:
```suggestion
<!-- # Copyright 2015-2016 Tecnativa - Jairo Llopis
```

File: website_snippet_big_button/templates/snippets.xml [hunk 7]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2015-2016 Jairo Llopis <jairo.llopis@tecnativa.com>
```
Comment (victoralmau) [inline website_snippet_big_button/templates/snippets.xml:2]:
```suggestion
<!-- Copyright 2015-2016 Jairo Llopis
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Good job, only some little changes and squash administrative commits too.

File: website_snippet_big_button/static/src/scss/website_snippet_big_button.scss [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+        height: 100%;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-around;
+
+        > .d-block {
+            white-space: normal;
+        }
+    }
+}
```
Comment (Tardo) [inline website_snippet_big_button/static/src/scss/website_snippet_big_button.scss:21]: Don't need this css rules
```suggestion
```

File: website_snippet_big_button/templates/snippets.xml [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html). -->
+<odoo>
+    <template id="s_big_button" name="Big Buttons">
+        <section class="oe_snippet_body big-buttons">
+            <div class="container">
+                <div class="row mt16 mb16">
+                    <div class="text-center col-sm-6 offset-lg-2 col-lg-4 pb8">
+                        <a
+                            href="/contactus"
+                            class="btn btn-success btn-lg font-weight-bold"
```
Comment (Tardo) [inline website_snippet_big_button/templates/snippets.xml:13]:
```suggestion
                            class="btn btn-success btn-lg font-weight-bold d-flex flex-column justify-content-around"
```

File: website_snippet_big_button/templates/snippets.xml [hunk 10]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2015-2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+     Copyright 2019 Tecnativa - Cristina Martin R.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html). -->
+<odoo>
+    <template id="s_big_button" name="Big Buttons">
+        <section class="oe_snippet_body big-buttons">
+            <div class="container">
+                <div class="row mt16 mb16">
+                    <div class="text-center col-sm-6 offset-lg-2 col-lg-4 pb8">
```
Comment (Tardo) [inline website_snippet_big_button/templates/snippets.xml:10]:
```suggestion
                    <div class="text-center justify-content-center col-sm-6 col-lg-4 pb8">
```

File: website_snippet_big_button/templates/snippets.xml [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+                            href="/contactus"
+                            class="btn btn-success btn-lg font-weight-bold"
+                        >
+                            <span class="h3 d-block">I want this</span>
+                            <i class="fa fa-thumbs-o-up fa-5x mt16 mb16" />
+                            <span class="d-block">Request budget</span>
+                        </a>
+                    </div>
+                    <div class="text-center col-sm-6 col-lg-4 pb8">
+                        <a href="/" class="btn btn-danger btn-lg font-weight-bold">
```
Comment (Tardo) [inline website_snippet_big_button/templates/snippets.xml:21]:
```suggestion
                        <a href="/" class="btn btn-danger btn-lg font-weight-bold d-flex flex-column justify-content-around">
```

**Review by Tardo — CHANGES_REQUESTED** [review]
It is not necessary to define your own classes for the styles being applied. So you can delete the .scss file and use bootstrap.

**The migration is accepted as the effort of doing the migration script is not worth it.**

**Review by Tardo — APPROVED** [review]

**Pull request merged**